src/server.ts

```diff
@@ -59,7 +60,7 @@ app.use(
 		resave: true,
 		saveUninitialized: true,
 		secret: 'dcOVe-ZW3ul77l23GiQSNbTJtMRio87G2yUOUAk_otcbL3uywfyLMZ9NBmDMuuOt',
-		store: sessionStore,
+		store: new FileStore({path: process.env.NOW ? `/tmp/sessions` : `.sessions`}),
```

Comment:
This should be coming from an Application Configuration rather than on the spot decision and initialisation